README.md

```diff
@@ -115,11 +115,11 @@ Next, we'll set up the endpoints for use with our API.
 
 ```swift
 public func url(route: MoyaTarget) -> String {
-    return route.baseURL.URLByAppendingPathComponent(route.path).absoluteString
+    return route.baseURL.URLByAppendingPathComponent(route.path).absoluteString!
 }
 
 let endpointsClosure = { (target: GitHub, method: Moya.Method, parameters: [String: AnyObject]) -> Endpoint<GitHub> in
-    return Endpoint<GitHub>(URL: url(target), method: method, parameters: parameters, sampleResponse: .Success(200, target.sampleData))
+    return Endpoint<GitHub>(URL: url(target),sampleResponse: .Success(200, target.sampleData), method: method, parameters: parameters )
```

Comment:
Very strange! The `Endpoint` initializer [has default values](https://github.com/ashfurrow/Moya/blob/f9ccd82cfe6135faeaa0f35247e00524beff4873/Endpoint.swift#L28) for those two parameters – was this not working for you?